spinnman/messages/eieio/eieio_command_message.py

```diff
@@ -9,7 +9,7 @@
 
 class EIEIOCommandMessage(AbstractEIEIOMessage):
 
-    def __init__(self, eieio_command_header, data):
+    def __init__(self, eieio_command_header, data=bytearray()):
```

Comment:
Can't do this as it will be the same bytearray().  Instead use None and then create a bytearray()!